datajunction-server/alembic/env.py

```diff
@@ -65,9 +64,9 @@ def run_migrations_offline():
     script output.
 
     """
-    url = settings.index
+    x_args = context.get_x_argument(as_dictionary=True)
```

Comment:
@sadath-12 this `-x uri=` param will let you run the migration script for different tenants.
```
alembic -x uri="postgresql+psycopg://username:password@server:5432/tenant1" upgrade head
alembic -x uri="postgresql+psycopg://username:password@server:5432/tenant2" upgrade head
# ...etc
```

...or in python

```py
from alembic.config import Config
from alembic import command

DATABASE = "tenant1"

alembic_cfg = Config()
alembic_cfg.set_main_option("script_location", "./alembic")
alembic_cfg.cmd_opts = {"x": [f"uri=postgresql+psycopg://username:password@server:5432/{DATABASE}"]}
command.upgrade(alembic_cfg, "head")
```